content/runbooks/kubernetes/CPUThrottlingHigh.md

```diff
@@ -0,0 +1,33 @@
+---
+title: CPU Throttling High
+weight: 20
+---
+
+# CPUThrottlingHigh
+
+## Meaning
+
+Given container in the pod is throttled to avoid excessive CPU usage.
```

Comment:
We should clearly state that this alert is just informative and user **shouldn't** increase CPU limits unless the application is behaving erratically (the is another alert firing)

Reply:
fixed